compose.yml

```diff
@@ -2,6 +2,7 @@ version: '3'
 
 services:
   mybot:
+    init: True
```

Comment:
Prefer `true`/`false` (notice the lowercase) to all the other boolean variants, as they are the most consistent with everything else in YAML.
Lowercase boolean values are typical around the development ecosystem. which is less surprising than `True`/`False`, `TRUE`/`FALSE`, `y`/`n`, `Y`/`No`, `on`/`off`, &c. (which all are valid booleans as per the spec.).